src/main/java/kenigsberg/transit/TransitFrame.java

```diff
@@ -55,10 +58,23 @@ public TransitFrame() {
         detailStopPanel.add(limitedInfoLabel);
 
 
-        controller = new TransitController(service, limitedInfoLabel, detailStopPanel);
+        DefaultListModel listModel = new DefaultListModel();
+
+        JList list = new JList(listModel);
```

Comment:
JTable, not JList

Reply:
I may have misspoke when we talked about this https://docs.oracle.com/javase/tutorial/uiswing/components/table.html